docs/flashbots-protect/rpc/fast.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Flashbots Protect Fast
```

Comment:
I think we should make sure to call it "Flashbots Protect Fast **Mode**." The words just flow better IMO. When talking about it with people, we'll probably be saying "fast mode" so I think we should match that in our docs.

I think "Fast Transactions" could be used for referring to transactions submitted via Fast Mode, but in referring to Fast Mode itself, I think it's important to stay consistent with the name.